`readability-inconsistent-declaration-parameter-name` in `str_format_opt`

```cpp
[7/216] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/csv.cpp.o
/home/solly/Code/ddnet/src/base/system.h:1283:12: warning: function template specialization 'str_format_opt<int>' has a primary template declaration with different parameter names [readability-inconsistent-declaration-parameter-name]
 1283 | inline int str_format_opt(char *buffer, int buffer_size, const char *format, int value)
      |            ^
/home/solly/Code/ddnet/src/base/system.h:1276:5: note: the primary template declaration seen here
 1276 | int str_format_opt(char *buffer, int buffer_size, const char *format, Args... args)
      |     ^
/home/solly/Code/ddnet/src/base/system.h:1283:12: note: differing parameters are named here: ('value'), in primary template declaration: ('args')
 1283 | inline int str_format_opt(char *buffer, int buffer_size, const char *format, int value)
      |            ^
```
Everytime str.h is included, I'm not sure what the error is

--- a/src/base/system.h
+++ b/src/base/system.h
@@ -1280,7 +1280,7 @@ int str_format_opt(char *buffer, int buffer_size, const char *format, Args... ar
 }
 
 template<>
-inline int str_format_opt(char *buffer, int buffer_size, const char *format, int val)
+inline int str_format_opt(char *buffer, int buffer_size, const char *format, int val) // NOLINT(readability-inconsistent-declaration-parameter-name)
 {
 	if(strcmp(format, "%d") == 0)
 	{

